com.visualstudio.code.yaml

```diff
@@ -27,6 +27,12 @@ finish-args:
   - --filesystem=xdg-config/kdeglobals:ro
   - --talk-name=com.canonical.AppMenu.Registrar
   - --talk-name=com.canonical.AppMenu.Registrar.*
+add-extensions:
+  com.visualstudio.code.tool:
+    directory: tools
```

Comment:
I would prefer the naming scheme from Atom:
`com.visualstudio.code.Extension`
`directory: extensions`

Reply:
IMO `Extension` suggests that it's VS Code extension, but it's not.

Reply:
OK, makes sense.